src/frontend/styles/vars.pcss

```diff
@@ -1,44 +1,70 @@
 :root {
   --color-text-main: #313649;
   --color-text-second: #5d6068;
-  --color-line-gray: #E8E8EB;
+  --color-line-gray: #e8e8eb;
   --color-link-active: #2071cc;
-  --color-link-hover: #F3F6F8;
+  --color-link-hover: #f3f6f8;
   --color-bg-light: #f8f7fa;
   --color-page-active: #ff1767;
 
-  --color-button-primary: #3389FF;
-  --color-button-primary-hover: #2E7AE6;
-  --color-button-primary-active: #296DCC;
+  --color-button-primary: #3389ff;
+  --color-button-primary-hover: #2e7ae6;
+  --color-button-primary-active: #296dcc;
 
   --color-button-secondary: #717682;
-  --color-button-secondary-hover: #5D6068;
-  --color-button-secondary-active: #4B4F5B;
-
-  --color-button-warning: #EF5C5C;
-  --color-button-warning-hover: #D65151;
-  --color-button-warning-active: #BD4848;
+  --color-button-secondary-hover: #5d6068;
+  --color-button-secondary-active: #4b4f5b;
 
+  --color-button-warning: #ef5c5c;
+  --color-button-warning-hover: #d65151;
+  --color-button-warning-active: #bd4848;
 
   /**
    * Site layout sizes
    */
   --layout-padding-horizontal: 22px;
   --layout-padding-vertical: 30px;
+
   --layout-sidebar-width: 344px;
+
   --layout-width-main-col: 650px;
   --layout-height-header: 56px;
 
+  --layout-max-width-aside-right: 450px;
+  --layout-max-space-from-right-aside: 160px;
+  --layout-available-right-aside-width: calc(
+    50% - var(--layout-width-main-col) / 2
+  );
+
+  --layout-width-aside-right: min(
+    var(--layout-available-right-aside-width),
+    var(--layout-max-width-aside-right)
+  );
+
+
+  @media (--extrem-wide-desktop) {
+    --layout-width-aside-right: calc(
+      var(--layout-available-right-aside-width)-var(
+          --layout-max-space-from-right-aside
+        )
+    );
+  }
+
+  @media (--tablet) {
+    --layout-width-aside-right: 0px;
```

Comment:
as `--tablet` media query includes screen widths [980px, 1050px], right column is still visible on mobiles
```
@custom-media --tablet all and (min-width: 980px) and (max-width: 1050px);
```

<img width="637" alt="image" src="https://user-images.githubusercontent.com/31101125/175521781-f70bc4d1-f169-454f-a2ff-562744bf9183.png">